README.md

```diff
@@ -101,7 +101,7 @@ The following document has been modified on 18th October 2018
 
 [Comparative Study of hydrophones used by other teams](https://github.com/auvzhcet/AUV2k19/blob/master/hydrophones_study.md)
 
-[Our implemetation](https://github.com/auvzhcet/Documentation/blob/master/Acoustics.md)
+[Our implemetation](docs/computer/Acoustics.md)
```

Comment:
Spelling mistake!

Reply:
I think it is correct as i have seen in preview and it is working

Reply:
"Implementation" it is

Reply:
ok